frontend/src/global_styles/content/_datepicker.sass

```diff
@@ -130,8 +132,8 @@ $datepicker--selected-border-radius: 5px
         max-width: 267px
 
       .flatpickr-day
-        color: $spot-color-basic-gray-1
-        background: $spot-color-basic-white
+        color: var(--body-font-color)
```

Comment:
This is indeed much darker in the normal mode now. That might be okay, we have to check with product or find an alternative